README.md

```diff
@@ -49,6 +49,27 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   roslaunch godel_surface_detection kinect2.launch
   ```
 
+### Applications network mode(Only simulation)
```

Comment:
It'd be nice to provide a brief description of what this "network" mode entails.